homeassistant/components/stream/worker.py

```diff
@@ -452,6 +452,10 @@ def stream_worker(
 ) -> None:
     """Handle consuming streams."""
 
+    if av.library_versions["libavformat"][0] >= 59:
```

Comment:
Today stimeout is always set by the caller, but it seems slightly more defensive to not assume that:

```suggestion
    if av.library_versions["libavformat"][0] >= 59 and "stimeout" in options:
```

Reply:
Sure